src/client/config/dev-cloud.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "useEndPoint": true,
```

Comment:
when you want to default to the same hostname as the application, i.e. location.hostname